components/brave_shields/resources/panel/container.tsx

```diff
@@ -6,50 +6,91 @@ import * as React from 'react'
 
 import MainPanel from './components/main-panel'
 import TreeList from './components/tree-list'
-import { ViewType } from './state/component_types'
+import {
+  MakeResourceInfoList,
```

Comment:
This should probably be `lowerCamelCase` too
```suggestion
  makeResourceInfoList,
```